docs/fargate.md

```diff
@@ -115,6 +120,93 @@ fi
 
 `EFS_ID` is going to be used in setting up Volumes for metric pods.
 
+## Logs
+
+The following are some of the steps needed to setup and enable logs collection on Fargate
+
+### Fluent-bit log router configuration
+
+Below are the steps to configure the log router on fargate
+
+- Create a dedicated Kubernetes namespace named `aws-observability`
+
+```yaml
+kind: Namespace
+apiVersion: v1
+metadata:
+  name: aws-observability
+  labels:
+    aws-observability: enabled
+```
+
+The fargate log router manages the `Service` and `Input` sections. One needs to create a `ConfigMap` to enable log routing to cloudwatch, an example is shown below:
+
+```yaml
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: aws-logging
+  namespace: aws-observability
+data:
+  flb_log_cw: "true"  # Set to true to ship Fluent Bit process logs to CloudWatch.
+  output.conf: |
+    [OUTPUT]
+        Name cloudwatch_logs
+        Match   kube.*
+        region region-code
+        log_group_name my-logs
+        log_stream_prefix from-fluent-bit-
+        log_retention_days 60
+        auto_create_group true
+```
+
+For more information on this refer to [fargate-logging](https://docs.aws.amazon.com/eks/latest/userguide/fargate-logging.html)
+
+### Cloudwatch logs collection
+
+After setting up fluent-bit to forward logs to cloudwatch, the next step is to setup and enable cloudwatch collection
+
+This involves the following:
+
+- [Enable cloudwatch collection](#enable-cloudwatch-collection)
+- Configure the `RELEASE-sumologic-otelcol-logs-collector` service account to assume an IAM role
+- Restart the pod `RELEASE-sumologic-otelcol-cloudwatch-collector-0`
+
+#### Enable cloudwatch collection (EKS fargate only)
+
+To enable cloudwatch log collection add the following to the `user-values.yaml`
+
+```yaml
+sumologic:
+  logs:
+    collector:
+      otelcloudwatch:
+        enabled: true
+```
+
+#### Configure the service account
+
+To configure the service account to use an IAM role use the command below
+
+```bash
+eksctl create iamserviceaccount --name release-sumologic-otelcol-logs-collector \
+                                --namespace default --cluster sumo-fargate \
+                                --role-name sumo-role \
+                                --attach-policy-arn arn:aws:iam::586421305253:policy/sumo-policy \
+                                --override-existing-serviceaccounts --approve
+```
```

Comment:
Where do we provide this `sumo-policy` and what is it going to be? IMO it should be just a policy that enables reading from cloudwatch

Reply:
Right the sumo-policy would get associated with the IAM service account. I linked the doc for creating the policy and the corresponding role in AWS in the doc
https://docs.aws.amazon.com/eks/latest/userguide/associate-service-account-role.html

Reply:
ok I can add the role permissions that are needed for the service account